src/Bicep.RegistryModuleTool/Bicep.RegistryModuleTool.csproj

```diff
@@ -2,16 +2,15 @@
 
   <PropertyGroup>

```

Comment:
>PropertyGroup [](http://example.com/codeflow?start=3&length=13)

Is nullability enabled at repo-level? I saw some projects enabling it explicitly but this one doesn't.

Reply:
Yes it's enabled in `Directory.Build.Props`. I forgot to remove `<Nullable>true</Nullable>` in the other two registry module projects which are added by VS. Will push a commit to remove them.